### PR TITLE
Changes the CMG to use 9mm peacekeeper instead of .45

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request
The NT CMG-1 is, essentially, no longer a straight rip of the c20 with slightly lowered firespeed. The ammo type has been moved from .45 to 9mm peacekeeper, which has lowered the fairly ridiculous firepower of a weapon we give to sec and the blueshield at roundstart.

 - .45mm rubbers were dealing 27 stamina damage a shot, which ignored armor, which is now 20 with 9mm PK
 - .45mm lethals did 30 damage a shot, which is now 15 with 9mm PK

There are also some fixes in here:

- We had two separate modular definitions for .45 rubber ammo, the non-gubman one was deleted
- We had an unused CMG type that was literally a subtype of the c20, which was deleted due to, again, being unused
- There were two definitions of a mag-less CMG in both CMG code and blueshield code, the one in CMG code was removed

## How This Contributes To The Skyrat Roleplay Experience
The CMG, no matter how you put it, is quite literally a straight rip of the nukeops c20r, with a slighty slower firerate (by 0.5) and a one round shorter burst fire. As a consequence of this, it is VERY strong, especially the rubber casings. Should the MCR rework I made go through, the CMG would be left up as another too powerful weapon, and so I figured I'd make all the weapon balance prs at around the same time so they all work together.

## Changelog

:cl:
balance: Due to lawsuits over the similarities in design between the CMG and C20r, the internal workings and round used in the CMG have been redesigned and re-issued to all stations
fix: Duplicate definitions of both .45 rubbers and magazine-less CMGs were dealt with
fix: A completely unused c20 subtype, also called the CMG, was gotten rid of due to being completely unused
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/guns/cmg.dm [hunk 1]
```diff
@@ -29,26 +29,25 @@
 		overlay_y = 10)
 
 /obj/item/ammo_box/magazine/multi_sprite/cmg
-	name = ".45 PDW magazine"
+	name = ".9 Peacekeeper PDW magazine"
```
Comment (tf-4) [inline modular_skyrat/modules/sec_haul/code/guns/cmg.dm]:
```suggestion
	name = "9mm Peacekeeper PDW magazine"
```

Comment (Paxilmaniac) [reply modular_skyrat/modules/sec_haul/code/guns/cmg.dm]: i fucke dup

**Comment by tf-4** [comment]
not really a fix

**Comment by Paxilmaniac** [comment]
yeah that's fair

File: modular_skyrat/modules/sec_haul/code/guns/cmg.dm [hunk 2]
```diff
@@ -5,14 +5,14 @@
  */
 
 /obj/item/gun/ballistic/automatic/cmg
-	name = "\improper NT CMG-1"
-	desc = "A bullpup two-round burst .45 PDW with an eerily familiar design. It has a foldable stock and a dot sight."
+	name = "\improper NT CMG-1-A1"
+	desc = "A bullpup, two round burst submachinegun chambered in 9mm peacekeeper that (after lawsuits from unnamed companies) now uses a fully unique design."
```
Comment (Draganfrukts) [inline modular_skyrat/modules/sec_haul/code/guns/cmg.dm]:
```suggestion
	desc = "A bullpup, two-round burst PDW chambered in 9mm Peacekeeper, developed by Nanotrasen R&D and based on a licensed Scarborough Arms design. It features a folding stock and comes pre-attached with a dot sight."
```

File: modular_skyrat/modules/sec_haul/code/guns/cmg.dm [hunk 3]
```diff
@@ -5,14 +5,14 @@
  */
 
 /obj/item/gun/ballistic/automatic/cmg
-	name = "\improper NT CMG-1"
-	desc = "A bullpup two-round burst .45 PDW with an eerily familiar design. It has a foldable stock and a dot sight."
+	name = "\improper NT CMG-1-A1"
```
Comment (Draganfrukts) [inline modular_skyrat/modules/sec_haul/code/guns/cmg.dm]:
```suggestion
	name = "\improper NT CMG-2 PDW"
```
Now it says it's a gun when you pull it out too, also avoids consistency issues

**Review by GoldenAlpharex — APPROVED** [review]
Looks good to me.

**Pull request merged**